src/consts.ts

```diff
@@ -12,11 +12,19 @@ export const RELAY_CHAIN_IDS = ['0'];
 /**
  * List of all known system parachains.
  */
-export const SYSTEM_PARACHAINS_NAMES = ['statemine', 'statemint', 'westmint'];
+export const SYSTEM_PARACHAINS_NAMES = [
```

Comment:
Lets add `asset-hub-polkadot`, `asset-hub-kusama`, and `asset-hub-westend` as well.